src/app/data-planes/components/RuleEntryList.vue

```diff
@@ -110,34 +97,33 @@
 </template>
 
 <script lang="ts" setup>
-import { KTable } from '@kong/kongponents'
-import { PropType } from 'vue'
-
 import AccordionItem from '@/app/common/AccordionItem.vue'
 import AccordionList from '@/app/common/AccordionList.vue'
 import CodeBlock from '@/app/common/CodeBlock.vue'
 import PolicyTypeTag from '@/app/common/PolicyTypeTag.vue'
-import TagList from '@/app/common/TagList.vue'
-import { RuleEntry, TableHeader } from '@/types/index'
-
-const tableHeaders: TableHeader[] = [
-  { label: 'Type', key: 'type' },
-  { label: 'Addresses', key: 'addresses' },
-  { label: 'Conf', key: 'config' },
-  { label: 'Origin policies', key: 'origins' },
-]
-
-const props = defineProps({
-  id: {
-    type: String,
-    required: false,
-    default: 'entry-list',
-  },
-
-  ruleEntries: {
-    type: Object as PropType<RuleEntry[]>,
-    required: true,
-  },
+import type { InspectRuleMatcher } from '@/types/index.d'
+import { useI18n } from '@/utilities'
```

Comment:
Could you import this from `@app/application` please? `@/utilities` will cease to exist pretty soon.